ui/src/components/createTournament/EntryRequirements.tsx

```
@@ -422,85 +424,96 @@ const EntryRequirements = ({ form }: StepProps) => {
+                                                    ? "bg-brand/70 text-black"
+                                                    : "bg-background hover:bg-brand/20"
+                                                } px-4 py-2 hover:cursor-pointer`}
+                                                onClick={() => {
+                                                  if (!beenSelected) {
+                                                    field.onChange([
+                                                      ...(field.value || []),
+                                                      tournament.tournament,
+                                                    ]);
+                                                  }
+                                                }}
+                                              >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)